packages/docusaurus-theme-classic/src/theme/DocItem/styles.module.css

```diff
@@ -30,6 +30,24 @@
   top: calc(var(--ifm-navbar-height) + 2rem);
 }
 
+.tableOfContents::-webkit-scrollbar {
```

Comment:
this is not supported by FireFox, maybe we can use [scrollbar color](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color)

Reply:
Its not supported by firefox but its supported in all other major browsers like Safari and Chrome. 

Scrollbar color is an experimental feature in Firefox >= 64 and all major browsers doesnt support that.

In fact, its a bug of mozilla. No scrollbar equivalent in mozilla https://stackoverflow.com/questions/6165472/custom-css-scrollbar-for-firefox

https://bugzilla.mozilla.org/show_bug.cgi?id=77790

Anyway, This is same behavior like v1, which is kinda battle tested. I think its ok if firefox wont see this